onnxruntime/core/providers/acl/nn/batch_norm.h

```diff
@@ -38,7 +38,7 @@ class BatchNorm final : public OpKernel {
     ORT_ENFORCE(st.IsOK(), st.ErrorMessage());
 
     provider_ = (const_cast<ACLExecutionProvider*>(
```

Comment:
Oh, we have been throught const_cast already.  Last time it required several days of debugging. Can you file a bug?

Reply:
Issue here or aiinfra?

Reply:
Done (internal bug 809210)